src/crm/components/CrmSelectCompany.tsx

```diff
@@ -33,7 +33,7 @@ export default function CrmSelectCompany() {
               fontSize="small"
               sx={{ color: "primary.main" }}
             />
-            Acme Corporation
+            hello korey
```

Comment:
@builderio-bot can you put this text into a constant and move it to another file and import it to use here